api/README.md

```diff
@@ -0,0 +1,137 @@
+# API
+
+This scripts are designed to simplify the deployment and configuration of an application using EzAppConfig and Helm charts.
+API folder contains the scripts and configuration files related to the EzAppConfig cli API.
+
+## Prerequisites
+
+Before running the scripts in this folder, ensure that the following tools are installed:
+
+- [helm](https://github.com/helm/helm/releases/tag/v3.14.2)
+- [jq](https://jqlang.github.io/jq/download/)
+- [yq](https://github.com/mikefarah/yq?tab=readme-ov-file#download-the-latest-binary)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+## Getting started
+
+Clone the repository:
+
+```console
+git clone git@github.com:HPEEzmeral/ezua-ezapp-demo-example.git
+```
+
+Navigate to the script directory:
+
+```console
+cd api/tooling/
+```
+
+Ensure that the required tools are installed by running the following command:
+
+```console
+./check_required_tools.sh
+```
+
+Run the folowing command to see usage help:
+
+```console
+./bootstrap.sh
+```
+
+You will see the following output
+
+```
+Checking required tools...
+- Helm version: v3.14.3
+- jq version: 1.6
+- yq version: v4.40.5
+- kubectl version: v1.29.3
+Required tools checked.
+Usage: ./tooling/bootstrap.sh <COMMAND> [ARGUMENTS]
+
+Commands:
+        install
+        upgrade
+        delete
+        build_and_upload_chart_bundle
+                Uploads the current chart version to chartmuseum
+        delete_chart_bundle [chart_version]
+                Deletes the chart version from chartmuseum, if chart_version is not provided, current version is passed
+```
+
+Before you begin installing the application, please read the [configuration](#configuration) section.
```

Comment:
general idea or steps:

How to install app

How to upgrade app

How to delete app

Optional/Troubleshooting for chartmuseum

how to upload chart

how to delete chart